apps/docs-website/docs/vscode-support.md

```diff
@@ -2,9 +2,11 @@
 sidebar_position: 12
 ---
 
-# 🚀 VSCode support
+# 🚀 VS Code support
 
-To make the [VSCode ESLint Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) work better with Sheriff we can enable a few settings. It's advisable to enable them at the workspace level, meaning in the root of the project at `.vscode/settings.json`
+To make the [VS Code ESLint Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
```

Comment:
👎
i dont think there should be a linebreak here

Reply:
Oh, must've missed that one.